_env.example

```diff
@@ -0,0 +1,43 @@
+# Environment variables for MySQL service
+# WordPress database information
+MYSQL_USER=wpuser
+MYSQL_PASSWORD=userpassword
+MYSQL_DATABASE=test
+# MySQL replication user, should be different from above
+MYSQL_REPL_USER=repluser
+MYSQL_REPL_PASSWORD=replpassword
+
+
+# Environment variables for backups to Manta
+MANTA_URL=
+MANTA_USER=
+MANTA_SUBUSER=
+MANTA_ROLE=
+MANTA_KEY_ID=
+MANTA_PRIVATE_KEY=
+MANTA_BUCKET=
+
+# please include the scheme http:// or https:// in the URL variable
+WORDPRESS_URL=http://localhost
+WORDPRESS_SITE_TITLE=test site
+WORDPRESS_ADMIN_EMAIL=username@domain.com
+WORDPRESS_ADMIN_USER=username
+WORDPRESS_ADMIN_PASSWORD=password
+WORDPRESS_ACTIVE_THEME=theme
+WORDPRESS_CACHE_KEY_SALT=some-unique-string
+#WORDPRESS_TEST_DATA=False
+
+# Wordpress unique salts - these must be unique for your install to ensure
+# login security. visit the following link to generate a set for your site and
+# include them here
+# https://api.wordpress.org/secret-key/1.1/salt/
+WORDPRESS_AUTH_KEY=
+WORDPRESS_SECURE_AUTH_KEY=
+WORDPRESS_LOGGED_IN_KEY=
+WORDPRESS_NONCE_KEY=
+WORDPRESS_AUTH_SALT=
+WORDPRESS_SECURE_AUTH_SALT=
+WORDPRESS_LOGGED_IN_SALT=
+WORDPRESS_NONCE_SALT=
+
+CONSUL=consul.svc.{your-account-uuid}.{target-data-center}.cns.joyent.com
```

Comment:
This needs to get written in the `configure.sh`

I noticed the following error:

2016/04/26 16:22:36 Failed to query wordpress: Get http://consul.svc.%7Byour-account-uuid%7D.%7Btarget-data-center%7D.cns.joyent.com:8500/v1/health/service/wordpress?passing=1: dial tcp: lookup consul.svc.{your-account-uuid}.{target-data-center}.cns.joyent.com: invalid domain name [<nil>]